shell/platform/windows/fixtures/main.dart

```diff
@@ -159,6 +159,30 @@ void enableLifecycleToFrom() async {
   });
 }
 
+@pragma('vm:entry-point')
+void sendCreationMethod() async {
+  // The platform view method channel uses the standard method codec.
+  // See https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/message_codecs.dart#L262
+  // for the implementation of the encoding and magic number identifiers.
+  final List<int> data = <int>[
+    // Method name
+    7, 'create'.length, ...utf8.encode('create'),
```

Comment:
For readability, could we create constants for these? Something like:

```dart
const int valueNull = 0;
const int valueInt32 = 3;
const int valueString = 7;
const int valueMap = 13;
```

Reply:
I like it. Even better:
```cc
enum class ValueType {
  kValueNull = 0,
  kValueInt32 = 3,
  ...
};
```

Reply:
@cbracken I think this appears applicable only for C++ code. For the dart file, the `const int`s may be the best we can do.

Reply:
Agh yes, not sure where my mind was at there. Dart [enhanced enums](https://dart.dev/language/enums#declaring-enhanced-enums) do provide a mechanism for this, but it's entirely up to you.